src/v/redpanda/admin/api-doc/usage.json

```diff
@@ -24,6 +24,39 @@
         }
     ],
     "models": {
+        "datalake_topic_usage": {
+            "type": "object",
+            "description": "Datalake feature usage metrics of a single topic with datalake enabled",
+            "properties": {
+                "topic_name": {
+                    "type": "string",
+                    "description": "Name of the topic."
+                },
+                "topic_revision": {
```

Comment:
Just curious how this ends up being used? Do callers typically account for different revisions?

Reply:
Got it, just want to make sure the cloud team isn't expecting exactly one topic in this response (and isn't e.g. ignoring one of them if there are multiple)